<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Retried</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="7">Failed suite [Chase Bank Regression]</th></tr>
<tr><td><a href="#t0">login (failed)</a></td><td class="num">0</td><td class="num attn">1</td><td class="num">0</td><td class="num attn">1</td><td class="num">212</td><td></td><td></td></tr>
<tr class="stripe"><td><a href="#t1">Deposite(failed)</a></td><td class="num">0</td><td class="num attn">1</td><td class="num">0</td><td class="num attn">1</td><td class="num">21</td><td></td><td></td></tr>
<tr><th>Total</th><th class="num">0</th><th class="num attn">2</th><th class="num">0</th><th class="num attn">2</th><th class="num">233</th><th colspan="2"></th></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Failed suite [Chase Bank Regression]</th></tr></tbody><tbody id="t0"><tr><th colspan="4">login (failed) &#8212; failed</th></tr><tr class="failedeven"><td rowspan="1">com.ChaseBankTestNgListeners.LoginTest</td><td><a href="#m0">submitTest</a></td><td rowspan="1">1622323140285</td><td rowspan="1">17</td></tr><tr><th colspan="4">login (failed) &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="1">com.ChaseBankTestNgListeners.LoginTest</td><td><a href="#m1">forgetLinkTest</a></td><td rowspan="1">1622323140337</td><td rowspan="1">0</td></tr></tbody>
<tbody id="t1"><tr><th colspan="4">Deposite(failed) &#8212; failed</th></tr><tr class="failedeven"><td rowspan="1">com.ChaseBankTestNgListeners.DepositeTest</td><td><a href="#m2">selectAmountTest</a></td><td rowspan="1">1622323140484</td><td rowspan="1">1</td></tr><tr><th colspan="4">Deposite(failed) &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="1">com.ChaseBankTestNgListeners.DepositeTest</td><td><a href="#m3">confrimAmountTest</a></td><td rowspan="1">1622323140492</td><td rowspan="1">0</td></tr></tbody>
</table>
<h2>login (failed)</h2><h3 id="m0">com.ChaseBankTestNgListeners.LoginTest#submitTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [5] but found [4]
	at com.ChaseBankTestNgListeners.LoginTest.submitTest(LoginTest.java:22)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">com.ChaseBankTestNgListeners.LoginTest#forgetLinkTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.Throwable: Method LoginTest.forgetLinkTest()[pri:3, instance:com.ChaseBankTestNgListeners.LoginTest@49fc609f] depends on not successfully finished methods
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 18 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h2>Deposite(failed)</h2><h3 id="m2">com.ChaseBankTestNgListeners.DepositeTest#selectAmountTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [4] but found [5]
	at com.ChaseBankTestNgListeners.DepositeTest.selectAmountTest(DepositeTest.java:15)
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m3">com.ChaseBankTestNgListeners.DepositeTest#confrimAmountTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.Throwable: Method DepositeTest.confrimAmountTest()[pri:1, instance:com.ChaseBankTestNgListeners.DepositeTest@10bbd20a] depends on not successfully finished methods
	at java.util.ArrayList.forEach(Unknown Source)
... Removed 18 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
